.pre-commit-config.yaml

```diff
@@ -87,7 +87,7 @@ repos:
         args:
           # Indent by two spaces
           - -i
-          - '2'
+          - "2"
```

Comment:
I would like this change undone. We can always punt it upstream if the team would prefer double to single quotes.

Reply:
@mcdonnnj resolved with [c26a7cc](https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/c26a7cc1a45ba8feff464e7b5c094f1aca496b34)

I missed this sorry, this was auto formatted in VS Code